background/get-addon-settings.js

```diff
@@ -46,6 +46,15 @@ chrome.storage.sync.get(["addonSettings", "addonsEnabled"], ({ addonSettings = {
         addonsEnabled["block-count"] = true;
       }
       if (addonId === "editor-dark-mode") {
+        if (settings.textShadow && addonsEnabled["custom-block-text"] === undefined) {
```

Comment:
`custom-block-text` comes before `dark-www` in addons.json, so there's no way it will be undefined at this point.